src/encoded/tests/test_permissions.py

```diff
@@ -272,10 +290,11 @@ def test_submission_center_can_edit_file(test_submission_center, submission_cent
         'obsolete',
         'archived',
         'deleted',
-        'public'
+        'public',
+        'restricted'
     ])
     def test_submission_center_cannot_edit_file(test_submission_center, submission_center_user_app, released_file,
```

Comment:
This test should probably be supplemented/duplicated for an app with a user of the same submission center but lacking the appropriate `submits_for` association to ensure no creation or edit.